### PR TITLE
gui, bugfix: fixup resource request on win32 webview.

### PR DESCRIPTION
**Pull request by richardo2016**
Please fill in this template.

- [x] Use a meaningful title for the pull request with prefix '(module|object|source_directory) [bugfix|feat|refactor|]', Such as `db, bugfix: ...`, `tools, feat: ...`
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `fibjs test`.)
- [x] Ensure commits history clean, merge master/dev and clean meaningless commits by `git rebase`.

### CODE REVIEW

File: fibjs/src/gui/windows/WebView.cpp [hunk 1]
```diff
@@ -99,7 +99,16 @@ class gui_worker : public IClassFactory {
             m_pProtSink = pIProtSink;
             m_pProtSink->AddRef();
 
-            result_t hr = fs_base::cc_openFile(utf16to8String(szUrl + 5), "r", m_file, s_isolates.head());
+            exlib::string origin(utf16to8String(szUrl));
+            if (!qstrcmp(origin.substr(0, 5).c_str(), "fs://")) {
```
Comment (xicilion) [inline fibjs/src/gui/windows/WebView.cpp:103]: 这里直接写
```
if (!qstrcmp(origin.c_str(), "fs://", 5))
```

File: fibjs/src/gui/windows/WebView.cpp [hunk 2]
```diff
@@ -99,7 +99,16 @@ class gui_worker : public IClassFactory {
...
+                origin = origin.substr(5, origin.length());
+
+                int len = origin.length();
+                if (len > 0 && origin[len - 1] == '/')
```
Comment (xicilion) [inline fibjs/src/gui/windows/WebView.cpp:107]: 这个判断应该挪到 fs:// 判断这个块的外面，因为非 fs:// url 也需要这个处理

Comment (richardo2016) [reply fibjs/src/gui/windows/WebView.cpp]: done

File: fibjs/src/gui/windows/WebView.cpp (hunk 1 above)
Comment (richardo2016) [reply fibjs/src/gui/windows/WebView.cpp]: done

**Pull request merged**